win64.mak

```diff
@@ -86,21 +86,21 @@ unittest : $(SRCS) $(DRUNTIME)
 CC32=$(CC)\..\..\cl
 
 druntime32mscoff:
-	$(MAKE) -f win64.mak "DMD=$(DMD)" MODEL=32mscoff "CC=\$(CC32)"\"" "AR=\$(AR)"\"" "VCDIR=$(VCDIR)" "SDKDIR=$(SDKDIR)"
+	$(MAKE) -f win64.mak "DMD=$(DMD)" MODEL=32mscoff "CC=$(CC32)" "AR=$(AR)" "VCDIR=$(VCDIR)" "SDKDIR=$(SDKDIR)"
```

Comment:
`$(MAKE)` doesn't need any quotes?

Reply:
It doesn't get overloaded as often, and not always forwarded when invoking another make. It's propably still better to quote it, too.